packages/workspace/src/tasks-runner/cache.ts

```diff
@@ -132,12 +135,12 @@ export class Cache {
   }
 
   copyFilesFromCache(
```

Comment:
could you restore the signature? This is a breaking change if you do it this way